lark/lark.py

```diff
@@ -1,14 +1,12 @@
 from __future__ import absolute_import
 
-
-from lark.exceptions import ConfigurationError, assert_config
-
+from abc import ABC, abstractmethod
 import sys, os, pickle, hashlib
 from io import open
```

Comment:
Not needed in python3 AFAIK.

Reply:
Why not?

Reply:
Nevermind, right, it's just an alias.

Reply:
https://docs.python.org/3/library/io.html#io.open